docs/source/guide/setup_project.md

```diff
@@ -117,27 +106,32 @@ If you want, you can allow empty annotations.
 
 ### Set up task sampling
 
-If you're using Label Studio Community Edition, you must set up task sampling when you start Label Studio. See [Set up task sampling for your project](start.html#Set-up-task-sampling-for-your-project).
-
+In Label Studio Community Edition, you can set up task sampling from the command line when you start Label Studio or from the Label Studio UI. 
+- To start a project with specific task sampling, see [Set up task sampling for your project](start.html#Set-up-task-sampling-for-your-project).
+- To change task sampling settings from the Label Studio UI, do the following:
+    1. Within a project on the Label Studio UI, click **Settings**.
+    2. On the **General** settings tab, under **Task Sampling**, choose between `Sequential sampling` and `Random sampling`. 
+    3. Click **Save**. 
+    
 <i class='ent'></i> In Label Studio Enterprise, you can set up task sampling in the annotation settings for a project.
 1. Within a project on the Label Studio UI, click **Settings**.
 2. Click **Annotation Settings**.
 3. Select your preferred method of task sampling:
-- Uncertainty sampling, where tasks are shown to annotators according to the model uncertainty, or prediction scores.
-- Sequential sampling, the default, where tasks are shown to annotators in the same order that they appear on the Data Manager.
-- Uniform sampling, where tasks are shown to annotators in a random order.
+    - Uncertainty sampling, where tasks are shown to annotators according to the model uncertainty, or prediction scores.
```

Comment:
Uncertainty sampling is Enterprise only feature

Reply:
yes, this section has the enterprise badge on it.